test/mocks/payment-request-data.js

```diff
@@ -34,6 +34,7 @@ const paymentRequestDataMock = {
             integerAmount: 10000,
         },
         taxTotal: {
+            amount: 10.0,
```

Comment:
Are these `amount` entries still required if we're using `integerAmount`?

Reply:
are not necessary.